web/src/main/webapp/WEB-INF/classes/setup/sql/data/data-db-default.sql

```diff
@@ -731,6 +731,7 @@ INSERT INTO Settings (name, value, datatype, position, internal) VALUES ('system
 INSERT INTO Settings (name, value, datatype, position, internal) VALUES ('system/security/passwordEnforcement/usePattern', 'true', 2, 12002, 'n');
 INSERT INTO Settings (name, value, datatype, position, internal, editable) VALUES ('system/security/passwordEnforcement/pattern', '^((?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*(_|[^\w])).*)$', 0, 12003, 'n', 'n');
 
+INSERT INTO Settings (name, value, datatype, editable, encrypted, position, internal) VALUES ('metadata/batchediting/accesslevel', '', 0, 'y', 'n', 180001, 'n');
```

Comment:
You also need to add an entry to the latest migration script so that users performing an upgrade will get the new default value.

The current version is 

https://github.com/geonetwork/core-geonetwork/blob/main/web/src/main/webapp/WEB-INF/classes/setup/sql/migrate/v440/migrate-default.sql

The entry could be similar to the one used by `metadata/import/userprofile`

https://github.com/geonetwork/core-geonetwork/blob/16f4c910b52dc494dfde9ab8a6c77b66561330bf/web/src/main/webapp/WEB-INF/classes/setup/sql/migrate/v407/migrate-default.sql#L9

Reply:
Added to the migration script